FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -27,7 +27,9 @@ using namespace xercesc;
 
 namespace {
   inline std::string _toString(XMLCh const* toTranscode) {
-      std::string tmp(XMLString::transcode(toTranscode));
+      char* localForm = XMLString::transcode(toTranscode);
```

Comment:
Let's use RAII to handle the memory. So you can do
```C++
using OwningXMLStringPtr = std::unique_ptr<char, XMLString::release >;
```

So the above code becomes
```C++
OwningXMLStringPtr localForm{ XMLString::transcode(toTranscode) } ;
return std::string( localForm.get() );
```
I'm not 100% sure that the compiler will accept the function pointer as a template argument. If not, then We'd just create a new class which held a `std::unique_ptr`
```C++
struct OwningXMLStringPtr {
   explicit OwningXMLStringPtr(char* iChar) : m_ptr{iChar, XMLString::release} {}
   char* get() { m_ptr.get(); }
   private:
      std::unique_ptr<char, decltype( XMLString::release )> m_ptr;
};
```